README.md

```diff
@@ -170,6 +198,39 @@ Accepts an `options` object as an argument. Here are the keys in the `options` o
 #### `strategy` - default (`'accented'`)
 The pseudo localization strategy to use when transforming strings. Accepted values are `accented` or `bidi`.
 
+## CLI Interface
+For easy scripting a CLI interface is exposed. The interface supports JSON files and JS modules.
+
+```bash
+npx pseudo-localize ./path/to/file.json
+
+# pass in a JS transpiled ES module or CJS module
+npx pseudo-localize ./path/to/file
```

Comment:
I'm in favour of supporting passing a json file or js object file but I think a CLI of `pseudo-localize` should support getting a simple string as an argument. For example

```
npx pseudo-localize "abc"
```

Reply:
Hmmmm, I think we can support a pipe like this:

```bash
echo foo | npx pseudo-localize
```

This should work on Windows and *nix systems. I'm not in favor of mixing types in `src`, but maybe there can be an `--input` option. How about:

```bahs
npx pseudo-localize -i foo # or 
npx pseudo-localize --input foo
```

Thoughts?

Reply:
yes sounds good